Source/Fuse.Reactive.JavaScript/ObservableProperty.uno

```diff
@@ -8,7 +8,7 @@ namespace Fuse.Reactive
 	{
 		public LazyObservableProperty(ThreadWorker w, Scripting.Object obj, Property p): base(w, obj, p)
 		{
-			w.Context.ObjectDefineProperty(obj, p.Name.ToString(), Get);	
+			w.Context.ObjectDefineProperty(obj, JSName, Get);	
```

Comment:
There is no point adding these unless we expect people to use it and make it the best practice. This means rewriting a lot of docs and ending up with uglier-looking code.